src/odemis/model/_metadata.py

```diff
@@ -194,3 +194,5 @@
 MD_GAIN_P = "Proportional gain"  # float
 MD_GAIN_I = "Integral gain"  # float
 MD_GAIN_D = "Derivative gain"  # float
+
+MD_EXTRA_SETTINGS = "Extra settings"
```

Comment:
Indicate the purpose and type: "string containing a dict encoded in JSON, which represents all the known state of all the hardware used during an acquisition"